cluster/addons/fluentd-elasticsearch/es-image/run.sh

```diff
@@ -14,9 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+export ES_VERSION='2.4.0'
 export NODE_MASTER=${NODE_MASTER:-true}
 export NODE_DATA=${NODE_DATA:-true}
-/elasticsearch_logging_discovery >> /elasticsearch-1.5.2/config/elasticsearch.yml
+export NODE_ENDPOINTS=$(curl -k -s -L --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_PORT_443_TCP_PORT/api/v1/namespaces/peng/endpoints?labelSelector="k8s-app=elasticsearch-logging" | jq -r '.items[].subsets[].addresses | map(.ip) | join(",")')
```

Comment:
* This set will change over time.  Will ES track the changing set?
* What these aren't up and running yet?  Do you want to wait around until they are?
* The namespace should come from the environment.  `peng` is hard coded here.